providers/dell/idrac8/configure.go

```diff
@@ -740,6 +741,30 @@ func (i *IDrac8) Network(cfg *cfgresources.Network) (reset bool, err error) {
 	}
 
 	i.log.V(1).Info("Network config parameters applied.", "IP", i.ip, "HardwareType", i.HardwareType())
+
+	// SNMP section
```

Comment:
I suggest this is added into its own function `func (i *IDrac9) SetSNMP(enable bool) error`

Reply:
Maybe better to make it private?

`func (i *IDrac9) setSNMP(enable bool) error` (lowercase s)

What do you think?